src/Renci.SshNet/Renci.SshNet.csproj

```diff
@@ -42,10 +42,10 @@
     <PackageReference Include="Nerdbank.GitVersioning" Version="3.7.70-alpha" PrivateAssets="all" />
   </ItemGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0' or '$(TargetFramework)' == 'netstandard2.1'">
```

Comment:
We don't want/need to reference Microsoft.Bcl.AsyncInterfaces for netstandard2.1. We should have:

net462 and netstandard2.0: AsyncInterfaces, Asn1, BouncyCastle
netstandard2.1: Asn1, BouncyCastle
netN.0: BouncyCastle

Reply:
Just out of curiosity,why `Microsoft.Bcl.AsyncInterfaces` targets netstandard2.1?

Reply:
So that it does not result in unnecessary transitive dependencies (like System.Threading.Tasks.Extensions). It is actually just a facade which forwards to the in-box types: https://github.com/dotnet/runtime/blob/main/src/libraries/Microsoft.Bcl.AsyncInterfaces/src/Microsoft.Bcl.AsyncInterfaces.csproj

Reply:
Yes, the `Microsoft.Bcl.AsyncInterfaces` for netstandard2.1 is just a facade, but `System.Formats.Asn1` for net6.0+ contains all types which is confused.
![image](https://github.com/user-attachments/assets/16959159-e6d2-4a06-954a-886240527274)
![image](https://github.com/user-attachments/assets/3f3f920d-6464-4a75-830d-b787190448b1)